blessings/__init__.py

```diff
@@ -94,8 +94,13 @@ def __init__(self, kind=None, stream=None, force_styling=False):
             # init sequences to the stream if it has a file descriptor, and
             # send them to stdout as a fallback, since they have to go
             # somewhere.
-            setupterm(kind or environ.get('TERM', 'unknown'),
-                      self._init_descriptor)
+            try:
+                setupterm(kind or environ.get('TERM', 'dumb'),
+                          self._init_descriptor)
+            except:
```

Comment:
Can we tighten this up to catch only the expected sorts of errors?

Reply:
ah, I was worried that the exception thrown (`_curses.error`) was private, but I think I can catch it with `curses.error` which is documented.